### PR TITLE
Updated uninstall hook. Delete all resources owned by the app

### PR DESCRIPTION
**Pull request by UnderGreen**

### CODE REVIEW

File: images/uninstall/uninstall.sh [hunk 1]
```diff
@@ -2,4 +2,15 @@
 
 kubectl label nodes -l pithos-role=node pithos-role-
 kubectl delete configmap cassandra-cfg pithos-cfg
-kubectl delete -f /var/lib/gravity/resources/pithos.yaml
+kubectl delete -f /var/lib/gravity/resources/cassandra.yaml
+kubectl delete -f /var/lib/gravity/resources/pithos-rc.yaml
+kubectl delete secret cassandra-ssl pithos-keys
+
+for sname in cluster-ca cluster-default-ssl cluster-kube-system-ssl
```
Comment (sofuture) [inline images/uninstall/uninstall.sh]: We shouldn't delete these things. They can potentially be created by other applications, so we should leave them alone, and simply use them if they already exist.

**Comment by sofuture** [comment]
Otherwise lgtm

**Comment by UnderGreen** [comment]
@sofuture fixed by 50eba69

**Comment by sofuture** [comment]
:shipit:

**Pull request merged**